src/detext/train/train.py

```diff
@@ -210,6 +210,8 @@ def serving_input_fn(hparams):
         "weight_placeholder", [], tf.float32, 'weight', hparams.feature_names, tf.constant([1.0], dtype=tf.float32))
     label_placeholder, label = train_helper.create_placeholder_for_ftrs(
         "label_placeholder", [None], tf.float32, 'label', hparams.feature_names)
+    task_id_placeholder, task_id = train_helper.create_placeholder_for_ftrs(
```

Comment:
Suggest adding the info of type and shape of task id in the PR context

Reply:
Thanks, I will do that. Also, reply to your general comments (which I was not able to reply directly) on the PR context, I'll update more details for what this implementation can vs cannot do. I'm also planning to update related documentation after resolving comments/questions from other reviewers.